@kargakis this change is causing unit test for replica_set to fail
```
E0129 01:29:34.587205   35088 util.go:45] Metric for replicaset_controller already registered
--- FAIL: TestPatchPodFails (0.00s)
	replica_set_test.go:181: Unexpected number of creates.  Expected 2, saw 0
```
Need to debug this more

--- a/test/e2e/garbage_collector.go
+++ b/test/e2e/garbage_collector.go
@@ -96,9 +96,9 @@ func newOwnerRC(f *framework.Framework, name string) *v1.ReplicationController {
 	}
 }
 
-// verifyRemainingDeploymentsAndReplicaSets verifies if the number of the remaining deployment
-// and rs are deploymentNum and rsNum. It returns error if the
-// communication with the API server fails.
+// verifyRemainingDeploymentsAndReplicaSets verifies if the number
+// of the remaining deployment and rs are deploymentNum and rsNum.
+// It returns error if the communication with the API server fails.
 func verifyRemainingDeploymentsAndReplicaSets(
 	f *framework.Framework,
 	clientSet clientset.Interface,

